tern/__main__.py

```diff
@@ -81,10 +81,13 @@ def do_main(args):
         if args.name == 'lock':
             drun.execute_dockerfile(args, True)
         elif args.name == 'report':
-            if args.layer or args.layer_inclusive:
-                raise NotImplementedError("This feature is not implemented yet!")
             if args.dockerfile:
-                drun.execute_dockerfile(args)
+                if (not args.layer) and (not args.layer_inclusive):
```

Comment:
small nit: I don't think we care about the `args.layer_inclusive` value here since it doesn't mean anything without the `args.layer value`.  If there is a `--layer-inclusive` option on the command line without `--layer` we just ignore it. So I think only checking `args.layer` would be sufficient for the logic here and looks a tiny bit cleaner.